courses/serializers.py

```diff
@@ -27,12 +29,23 @@ def get_programpage_url(self, program):
         except ProgramPage.DoesNotExist:
             return
 
+    def get_enrolled(self, program):
+        """
+        Returns true if the user
+        """
+        if 'request' in self.context:
+            user = self.context['request'].user
```

Comment:
is it possible for this serializer to be called when there is a `request`, but no `user` attached to that request? like from a CMS page before the user logs in?